src/subscription_manager/managercli.py

```diff
@@ -1210,6 +1210,7 @@ def _do_command(self):
         restart_virt_who()
 
         print((_("The system has been registered with ID: %s ")) % (consumer_info["uuid"]))
+        print (_("The registered system name is: %s ")) % (consumer_info["consumer_name"])
```

Comment:
Please remove the trailing space after `%s`.

Parentheses are a little out of order. Should be
```python
print(_("The registered system name is: %s") % consumer_info["consumer_name"])
```

Please notice the parentheses are different - with what was there it was being interpreted as
```python
(
    print(_("The registered system name is: %s"))  # statement result: None
    %
    consumer_info["consumer_name"]  # statement result: hostname
)  # unsupported operand type(s) for %: 'NoneType' and 'unicode'
```
(the print was happening before interpolation), rather than
```python
print(
    _("The registered system name is: %s")
    %
    consumer_info["consumer_name"]
)
```
where the print happens after interpolation, as intended.